sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/LeaseStoreManagerImpl.java

```diff
@@ -136,14 +136,6 @@ public LeaseStoreManagerBuilderDefinition hostName(String hostName) {
             this.settings.getLeaseCollectionLink(),
             this.requestOptionsFactory);
 
-        if (this.settings.getLeaseCollectionLink() == null) {
```

Comment:
@milismsft to review change feed related change.

Reply:
I don't believe removing this is correct; we do not want a null pointer exception but an illegal argument exception if the lease container is not specified during the build.

What does spotbugs say that we needed this fixed?

Reply:
Just sync'ed with Kushagra, I guess since the check is already done above and that requestOptionFactory is always created, it's ok to remove the lines below

Reply:
As mentioned @milismsft this is just a duplicate condition, which is already present 10-15 lines above this code piece .. so removing it :)